views/layout-designer-page.twig

```diff
@@ -137,38 +169,37 @@
                         {% set previewCss = "" %}
                     {% endif %}
 
-                    <div id="region_{{ region.regionId }}"
-                         regionEnabled="{{ currentUser.checkEditable(region) }}"
-                         regionid="{{ region.regionId }}"
-                         layoutid="{{ layout.layoutId }}"
-                         zindex="{{ region.zIndex }}"
-                         tip_scale="1"
-                         designer_scale="{{ designerScale }}"
-                         width="{{ region.width }}"
-                         height="{{ region.height }}"
-                         href="{{ urlFor("region.timeline.form", {id: region.regionId}) }}"
-                         {% if currentUser.checkEditable(region) %}ondblclick="XiboFormRender($(this).attr('href'))"{% endif %}
-                         class="{{ disabledCss }} {{ previewCss }}"
-                         data-preview-url="{{ urlFor("region.preview", {id: region.regionId}) }}"
-                         style="position:absolute; width:{{ regionWidth }}px; height:{{ regionHeight }}px; top: {{ regionTop }}px; left:{{ regionLeft }}px; z-index: {{ region.zIndex }};">
+                    <div id="region_{{ region.regionId }}" regionEnabled="{{ currentUser.checkEditable(region) }}" regionid="{{ region.regionId }}" layoutid="{{ layout.layoutId }}" zindex="{{ region.zIndex }}" tip_scale="1" designer_scale="{{ designerScale }}" width="{{ region.width }}" height="{{ region.height }}" href="{{ urlFor("region.timeline.form", {id: region.regionId}) }}" {% if currentUser.checkEditable(region) %}ondblclick="XiboFormRender($(this).attr('href'))"{% endif %}
+                         class="{{ disabledCss }} {{ previewCss }}" data-preview-url="{{ urlFor("region.preview", {id: region.regionId}) }}" style="position:absolute; width:{{ regionWidth }}px; height:{{ regionHeight }}px; top: {{ regionTop }}px; left:{{ regionLeft }}px; z-index: {{ region.zIndex }};">
                         <div class="regionTransparency {{ transparencyCss }}" style="width:100%; height:100%;"></div>
 
                         {% if currentUser.checkEditable(region) %}
                             <div class="btn-group regionInfo pull-right">
                                 <button class="btn dropdown-toggle" data-toggle="dropdown">
-                                    <span class="region-tip">{{ region.width|round }} x {{ region.height|round }} ({{ region.left|round }},{{ region.top|round }})</span>
-                                    <span class="caret"></span>
+                                    <span class="region-tip">{{ region.width|round }} x {{ region.height|round }}
```

Comment:
i'm not sure the carriage returns here will make any difference to the output - but it might be better to have it on one line like before